parts/chapters/subsections/5.3/BRINE.fodt

```diff
@@ -4737,15 +4737,15 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
       <table:covered-table-cell/>
      </table:table-row>
     </table:table>
-    <text:p text:style-name="P29">Table <text:sequence text:ref-name="refTable0" text:name="Table" text:formula="ooow:Table+1" style:num-format="1">5.6</text:sequence>: BRINE Keyword Description</text:p>
+    <text:p text:style-name="P29">Table <text:sequence text:ref-name="refTable0" text:name="Table" text:formula="ooow:Table+1" style:num-format="1">5.6</text:sequence>: <text:a xlink:href="#__RefHeading___Toc162083_289573908">BRINE</text:a> Keyword Description</text:p>
     <text:p text:style-name="P64"/>
-    <text:p text:style-name="P64">See also the PRECSALT and VAPWAT keywords in the RUNSPEC section that activates OPM Flow’s Salt Precipitation model, and the PVTWSALT keyword in the PROPS section to define the water properties with respect to salt concentration.</text:p>
+    <text:p text:style-name="P64">See also the <text:a xlink:href="#__RefHeading___Toc332782_3149455253">PRECSALT</text:a> and <text:a xlink:href="#__RefHeading___Toc317543_3149455253">VAPWAT</text:a> keywords in the <text:a xlink:href="#__RefHeading___Toc55591_1778172979">RUNSPEC</text:a> section that activates OPM Flow’s Salt Precipitation model, and the <text:a xlink:href="#__RefHeading___Toc331848_501926209">PVTWSALT</text:a> keyword in the <text:a xlink:href="#__RefHeading___Toc39329_784232322">PROPS</text:a> section to define the water properties with respect to salt concentration.</text:p>
     <text:h text:style-name="Heading_20_4" text:outline-level="4"><text:bookmark-start text:name="__RefHeading___Toc715589_3964674244"/>Example<text:bookmark-end text:name="__RefHeading___Toc715589_3964674244"/></text:h>
     <text:p text:style-name="P64">The first example actives the standard Brine model and has no terminating “/”.</text:p>
     <text:p text:style-name="_40_Example">--</text:p>
-    <text:p text:style-name="_40_Example">-- <text:s text:c="6"/>ACTIVATE STANDARD BRINE MODEL IN THE RUN</text:p>
+    <text:p text:style-name="_40_Example">-- <text:s text:c="6"/>ACTIVATE STANDARD <text:a xlink:href="#__RefHeading___Toc162083_289573908">BRINE</text:a> MODEL IN THE RUN</text:p>
     <text:p text:style-name="_40_Example">--</text:p>
-    <text:p text:style-name="_40_Example">BRINE</text:p>
```

Comment:
@hakonhagland, it looks like a link has been created here possibly because the style-name was directly specified as "_40_Example" rather than indirectly as for example "P64".